v4-client-py-v2/pyproject.toml

```diff
@@ -28,6 +28,7 @@ pytest-asyncio = "^0.23.6"
 python-dotenv = "^1.0.1"
 twine = "^4.0.2"
 black = "^24.4.0"
+ecdsa = "^0.19.0"
```

Comment:
is there a way to remove this dependency entirely?

Reply:
Definitelly 👍 It's removed in c9d92f1301a243b7dced382f15a0a3ffc648a8ec

For clarity `ecdsa` is no longer direct dependency but still will be pulled as `bip-utils` depends of it.
Based on [Snyk Advisor](https://snyk.io/advisor/python/bip-utils), `bip-utils` does not have these vulnerabilities and is considered secure.